depends/patches/systemtap/fix_variadic_warning.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/includes/sys/sdt.h b/includes/sys/sdt.h
+index 4075a5f..7c6138c 100644
+--- a/includes/sys/sdt.h
++++ b/includes/sys/sdt.h
+@@ -278,7 +278,7 @@ __extension__ extern unsigned long long __sdt_unsp;
+   _SDT_ASM_1(.purgem _SDT_TYPE_)					      \
+   _SDT_ASM_1(.purgem _SDT_TYPE)
+ 
+-#define _SDT_ASM_BODY(provider, name, pack_args, args, ...)		      \
++#define _SDT_ASM_BODY(provider, name, pack_args, args)		            \
```

Comment:
Add a comment that this patch can be dropped after switching to [C++20](https://github.com/bitcoin/bitcoin/issues/26916#issuecomment-1400239451)?

Reply:
Added.